src/dotnet/Chat.UI.Blazor/Pages/ChatPage.razor

```diff
@@ -14,8 +14,22 @@
     // Log.LogDebug("Rendered for #{ChatId}", chat.Id);
 }
 
+@if (ReferenceEquals(chat, SpecialChat.Unavailable) && m.IsInaccessiblePlace) {
+     <RenderIntoSlot Name="MainLeftDrawer">
```

Comment:
This also does not look good.
I think we should store in services not only selected chat but also chat or chat context and then react on their changes in left panel component.
We should discuss this.

Reply:
I answered above.